server/src/main/java/org/apache/druid/segment/loading/SegmentLoaderConfig.java

```diff
@@ -36,6 +36,9 @@
   @NotEmpty
   private List<StorageLocationConfig> locations = null;
 
+  @JsonProperty("lazyLoadOnStart")
+  private boolean lazyLoadOnStart = false;
```

Comment:
Please add documentation for this config . In the doc
- note in what situation it might be useful to set this option to true.
- note the catch mentioned in https://github.com/apache/incubator-druid/pull/6988#issuecomment-463528940 .

Reply:
docs added